README.md

```diff
@@ -1,29 +1,82 @@
 ![test workflow](https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/actions/workflows/tests.yml/badge.svg?branch=main)
 ![quality workflow](https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/actions/workflows/code-quality.yml/badge.svg?branch=main)
+
 # ScyllaDB JS Driver
 
-## Instalation and build process
+## Installation and build process
 
-For instalation type command below:
+For installation of the required packages, type command below:
 
-```
-$ npm install
+```bash
+npm install
 ```
 
 For build process use this command:
 
+```bash
+npm run build
+```
+
+If you want to build in development mode use this command:
+
+```bash
+npm run build:debug
+```
+
+If command above don't work, make sure that you have napi package downloaded globally.
+
+## Testing the driver
+
+There are three category of tests in this repository (with the corresponding commands to rn them):
+
+- Unit tests (``npm run unit``)
+- Integration short (``npm run integration_short``)
+- Integration long (``npm run integration_long``)
+
+The split between integration short and long comes from the datastax integration tests and remains currently unchanged.
+
+### Requirements
+
+Before running any of the tests, ensure the driver is build correctly.
+
+For the integration tests you need to do the following steps before running tests:
+
+1. Install [scylla-ccm](https://github.com/scylladb/scylla-ccm) package.
+You may also use [ccm](https://github.com/riptano/ccm) but not all tests are guaranteed to pass while using it.
+2. Have ``java-8`` installed and available in path ``/usr/lib/jvm/java-8`` (scylla-ccm uses this hardcoded path)
+3. Have ``simulacron.jar`` downloaded in ``SIMULACRON_PATH`` path (if you don't provide this env variable, home directory is assumed)
+You can download the latest version from the [repository](https://github.com/datastax/simulacron)
+
+## Running examples
+
+Before running examples, you need to build the driver and install packages for the examples by running:
+
+```bash
+npm install
 ```
-$ npm run build
+
+in ``./examples`` directory. You need also Cassandra or ScyllaDB running with at least three nodes (to ensure the replication_factor can be satisfied). By default, examples assume the entrypoint for the database is ``172.17.0.2:9042`` but you can change it by setting ``SCYLLA_URI`` env variable.
+
+You can run all examples with the following command:
+
+```bash
+npm run examples
 ```
 
-If command above don't work, make sure that you have napi package downloded globally.
+This will check if all examples finished successfully, but will hide any output generated by the example.
+
+Alternatively you can run just a specific example by providing the path to it:
+
+```bash
+node ./examples/basic.js
+```
 
 ## File structure
 
-| File / Folder     | Origin                             | Usage                                 
+| File / Folder     | Origin                             | Usage
```

Comment:
Nit: Folder -> Directory (the difference is subtle, but we are Linux chads, not Windows virgins, so let's use Linux vocabulary :cowboy_hat_face: ).